File: java/src/main/java/id/my/niznet/tugas2/Lingkaran.java
package id.my.niznet.tugas2;

public class Lingkaran implements BangunDatar {
    private double jariJari;

    public Lingkaran(double jariJari) {
        this.jariJari = jariJari;
    }

    @Override
    public double luas() {
        return Math.PI * jariJari * jariJari;
    }

    @Override
    public double keliling() {
        return 2 * Math.PI * jariJari;
    }
}

File: kotlin/src/main/kotlin/id/my/niznet/tugas2/Lingkaran.kt
package id.my.niznet.tugas2

class Lingkaran(private val jariJari: Double) : BangunDatar {
    override fun luas(): Double {
        return Math.PI * jariJari * jariJari
    }

    override fun keliling(): Double {
        return 2 * Math.PI * jariJari
    }
}